src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -17,6 +17,7 @@
 package com.pinterest.secor.common;
 
 import com.google.api.client.repackaged.com.google.common.base.Strings;
+import com.pinterest.secor.uploader.Uploader;
```

Comment:
Doesn't quite like this added dependency to secor.uploader package, I think you can have this property have a default value in secor.common.properties (there are quite a few config property specifying classname in secor.common.properties already)